spring-rabbit-stream/src/main/java/org/springframework/rabbit/stream/producer/ProducerCustomizer.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.rabbit.stream.producer;
+
+import java.util.function.BiConsumer;
+
+import com.rabbitmq.stream.ProducerBuilder;
+
+/**
+ * Called to enable customization of the {@link ProducerBuilder} when a new producer is
+ * created. The first parameter is the creator's bean name.
```

Comment:
"Creator"?
Many you mean a `ProducerBuilder` bean name?
Otherwise it is confusing who is "creator"...

Reply:
I was trying not to refer to the template since this can be reused by any component that creates a producer from the environment.

Maybe "The first parameter should be the bean name of the component that calls the customizer." ??

Reply:
Yes, this makes more sense.
Thanks